WebHostLib/templates/header/baseHeader.html

```diff
@@ -17,6 +17,8 @@
             </div>
             <div id="base-header-popover-menu">
                 <a href="/games">supported games</a>
+                <a href="/hint_games">hint games</a>
+                <a href="/tools">tools</a>
```

Comment:
Do we want to make this conditional on having at least one tool present?  Right now with no custom worlds, this links to an empty page.

Reply:
Would be sensible, I am unsure if I know how to do that.
Was hoping either someone else would, or we'd have a tool wanting to add itself to this as part of the PR so it wouldn't be blank.